### DISCUSSION TITLE
Logs going to cloudwatch->Splunk are not having task IDs.

### DISCUSSION THREAD

**Discussion by nikhilkarve** [apache/airflow#42723] [General]
### Description

Hello everyone.

We are sending our logs to Splunk through Cloudwatch. But when logs reach Splunk, only the messages are printed and not the task_id or the dag_id anything to identify where those logs are coming from. 

### Use case/motivation

I am thinking this would make remote logging more robust and descriptive for monitoring purposes.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by gopidesupavan** [comment]
It looks like issue in your cloudwatch to splunk configs might be your index config or field extractors in your splunk. 
Do you see the task_id/dag_ids appear in cloudwatch logs?

Are you using AWS MWAA?

**Comment by potiuk** [comment]
This is more of a discussion than concrete issue.
